etna/datasets/tsdataset.py

```diff
@@ -679,6 +679,25 @@ def regressors(self) -> List[str]:
         """
         return self._regressors
 
+    # TODO: is it ok that it includes target components and prediction intervals?
+    # TODO: don't we want to rename it into "variables" or smth like this for example?
```

Comment:
This needs a discussion

Reply:
Instead of simply `features`? Wouldn't it introduce confusion? Since we use this name, for example, the corresponding multi-index level is named 'feature'.

Reply:
- I agree that level "feature" is an argument in favor of "features"
- I think that some users could find it confusing the features includes target, components, etc. I found it during reworking 207 notebook (we compute number of features and it is strange that we should subtract 1).

Reply:
Decided to keep as is.